calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,168 @@
+| `max_initial_calibration_translation_error` | `double`      | `1.0`                                                   | Maximum allowable translation error in meters in the calibration process. If this error exceeds the specified value, a warning message will appear in the console. |
+| `max_initial_calibration_rotation_error`    | `double`      | `45.0`                                                  | Maximum allowable rotation error in degrees in the calibration process. If this error exceeds the specified value, a warning message will appear in the console.   |
+| `max_number_of_combination_samples`         | `int`         | `10000`                                                 | Maximum number of samples from combinations that are used for cross-validation during the calibration process.                                                     |
+
+## Requirements
+
+### Radar reflector
+
+The type of reflector shown in the image below is crucial for our calibration because it has a highly reliable and consistent response to radar waves. The triangular shape, often composed of three metal plates arranged in a prism form, ensures that the reflector returns signals in specific, predictable ways.
+
+It is recommended that the user build the radar reflector on a tripod and ensure it remains stable and secure. Additionally, nothing should be attached above the radar reflector; it must be the highest object on the entire calibration target. Furthermore, make sure the height of the radar reflector does not exceed the `reflector_max_height` parameter.
+
+<p align="center">
+    <img src="../docs/images/marker_radar_lidar_calibrator/radar_reflector.png" alt="radar_reflector" width="150">
+<p align="center">
+
+## Known issues/limitations
+
+- While extracting the background model, ensure that the radar reflectors are not in the calibration area and that no one is moving around in the area.
+
+- The calibrator provides a button to delete any mismatched pairs (e.g., an object detected by both radar and lidar). However, some outliers may not be easily detectable by human eyes, leading to imperfect results as the calibration proceeds even with these anomalies present. Future enhancements will aim to improve outlier detection, thereby refining the calibration accuracy.
+
+- The calibrator should be able to handle different lidar and radar sensors. So far, We calibrated the Velodyne VLS-128 lidar sensor, Pandar-40P lidar sensor, and ARS408 radar sensor with good calibration results.
+
+## Pro tips/recommendations
+
+- During calibration, all radar reflectors and the radar itself must lie on the same plane. This requirement does not apply to the lidar.
+- Position the reflectors at various distances, and ensure the center of each radar reflector always faces the radar sensor.
```

Comment:
Similar comment before. Various distances is not enough

Reply:
Is this the same line as before?
fixed in https://github.com/tier4/CalibrationTools/commit/2d5b949782c0793423b89ffb3fa0ca28e61b5296